dialme/README.md

```diff
@@ -30,7 +30,7 @@ This is a rather simple protocol that communicates a multiaddress to perform the
 message DialMe {
   message Peer {
     required bytes id = 1;    // peer id
```

Comment:
I wouldn't include this, the remote side should already *know* our peer ID and we definitely don't want to end up dialing a different peer.

Reply:
Hmm... I agree with you, I can't think of a use for this honestly. Unless anyone objects I'll remove it.